universal-application-tool-0.0.1/app/export/ExporterFactory.java

```diff
@@ -0,0 +1,31 @@
+package export;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import javax.inject.Inject;
+import models.Program;
+import services.program.ExportDefinition;
+
+public class ExporterFactory {
+  @Inject
+  public ExporterFactory() {
+    super();
+  }
+
+  public List<Exporter> createExporters(Program program) {
+    ImmutableList.Builder<Exporter> list = new ImmutableList.Builder<Exporter>();
+    for (ExportDefinition e : program.getProgramDefinition().exportDefinitions()) {
```

Comment:
nit: please avoid single-letter var names except for extremely common idioms like i for an index or e for an exception

Reply:
Done.